### ISSUE TITLE
Zone Volume < 0 fatal error is harsh - seems it should throw error but set it to some value like 10m3 and continue

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6266]
Issue overview
--------------
While testing a defect file, it now fails due to a fatal error.
```
   ** Severe  ** Indicated Zone Volume <= 0.0 for Zone=GARAGE PARKING
   ** Severe  ** All ZONE Volumes must be > 0.0
   **  Fatal  ** Errors found in Building Input, Program Stopped
```
There is the potential that many files that have less-than-perfect geometry will now fail where they ran before.  Seems the error should be severe or warning, but not fatal.  The zone volume could be set to something reasonable and let the simulation proceed.

This same file trips on this `assert( n == 4u );` in [DataSurfaces::Surface2D](https://github.com/NREL/EnergyPlus/blob/d945c222ed09925ab3f14e43ae76d990cbade53c/src/EnergyPlus/DataSurfaces.cc#L517).

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus Version 8.7.0-2dd86cbb4d

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added EnergyPlusDevSupport\DefectFiles
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by JasonGlazer** [comment]
@mjwitte please send me the file that causes this problem.

**Comment by Myoldmopar** [comment]
Closed via #6299
